src/services/notehub/AxiosHttpNotehubAccessor.ts

```diff
@@ -42,10 +42,22 @@ export default class AxiosHttpNotehubAccessor implements NotehubAccessor {
     };
   }
 
+  async getAllDevices(deviceUIDs: string[]) {
+    return Promise.all(deviceUIDs.map((device) => this.getDevice(device)));
+  }
+
   // Eventually we’ll want to find all valid gateways in a Notehub project.
-  // For now, just take the hardcoded gateway UID from the starter’s
+  // For now, just take the hardcoded list of gateway UID from the starter’s
   // environment variables and use that.
   async getDevices() {
+    // for retrieving multiple devices
+    if (this.hubDeviceUID.includes(",")) {
```

Comment:
Trivial but I think this `if` check is unnecessary—aka `split()` will still do what you want even if there is no `,` in the string.